Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Start with empty ubuntu machine
-FROM ubuntu:15.04
+FROM ubuntu:18.04
```

Comment:
Could this work with Ubuntu 20.04 LTS? I think using 20.04 now will save us a fair bit of trouble down the road when 18.04 reaches EOL

Reply:
I've run into issues before where some packages I need were not actually available on Ubuntu 20.04 yet as it's relatively new (released April this year). 18.04 will be supported until 2023 - I think we can consider upgrading a little further down the road once the 20.04 ecosystem stabilises